### PR TITLE
DSMYSQL-1561 API Docs update for HA upgrade functionality + DSMYSQL-1559 API Docs update for option to always do full backup for scheduled backup

### PR DESCRIPTION
**Pull request by ayersek64**

### CODE REVIEW

**Comment by rackernexus** [comment]
Build 83 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-c1efe9d35/docs/cloud-databases/v1/). valid for 14 days

File: api-docs/api-reference/methods/put-update-schedule-for-backups-by-schedule-id-version-accountid-schedules-scheduleid.rst [hunk 1]
```diff
@@ -37,6 +38,10 @@ schedule for backups by schedule ID.
     |                          |keep.                    |                         |
     +--------------------------+-------------------------+-------------------------+
 
+The ``day_of_week`` attribute specifies the day in which a full backup will be
+made. When ``day_of_week`` is 0-6, after that day, the schedule will automatically
```
Comment (catlook) [inline api-docs/api-reference/methods/put-update-schedule-for-backups-by-schedule-id-version-accountid-schedules-scheduleid.rst:42]: made. When ``day_of_week`` is 0-6, after that day, the schedule automatically

File: api-docs/api-reference/methods/put-update-schedule-for-backups-by-schedule-id-version-accountid-schedules-scheduleid.rst [hunk 2]
```diff
@@ -37,6 +38,10 @@ schedule for backups by schedule ID.
...
+run daily incremental backups until the next full backup.
```
Comment (catlook) [inline api-docs/api-reference/methods/put-update-schedule-for-backups-by-schedule-id-version-accountid-schedules-scheduleid.rst:43]: runs daily incremental backups until the next full backup.

File: api-docs/api-reference/methods/post-create-scheduled-backup-version-accountid-schedules.rst [hunk 3]
```diff
@@ -50,8 +51,8 @@ Backup:
     +--------------------------+-------------------------+-------------------------+
 
 The ``day_of_week`` attribute specifies the day in which a full backup will be
-made. After that day, the schedule will automatically run daily incremental
-backups until the next full backup.
+made. When ``day_of_week`` is 0-6, after that day, the schedule will automatically
```
Comment (catlook) [inline api-docs/api-reference/methods/post-create-scheduled-backup-version-accountid-schedules.rst:54]: made. When ``day_of_week`` is 0-6, after that day, the schedule automatically

File: api-docs/api-reference/methods/post-create-scheduled-backup-version-accountid-schedules.rst [hunk 4]
```diff
@@ -50,8 +51,8 @@ Backup:
...
+run daily incremental backups until the next full backup.
```
Comment (catlook) [inline api-docs/api-reference/methods/post-create-scheduled-backup-version-accountid-schedules.rst:55]: runs daily incremental backups until the next full backup.

**Review by catlook — CHANGES_REQUESTED** [review]
minor tweaks

**Comment by catlook** [comment]
@ayersek64 We wrapping this one up today?

**Comment by ayersek64** [comment]
Absolutely! <grin>
Just submitted updates for your comments.

**Review by catlook — APPROVED** [review]
👍

**Comment by rackernexus** [comment]
Build 84 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-21dbb07ef/docs/cloud-databases/v1/). valid for 14 days

**Pull request merged**